src/components/MenuBar/MenuBarToggleButton.tsx

```diff
@@ -0,0 +1,60 @@
+import {
+    IconButton,
+    IconButtonProps,
+    Tooltip,
+    TooltipTypeMap,
+} from "@mui/joy";
+
+import Check from "@mui/icons-material/Check";
+import Close from "@mui/icons-material/Close";
```

Comment:
to follow the convention in mui official docs:
```suggestion
import CheckIcon from "@mui/icons-material/Check";
import CloseIcon from "@mui/icons-material/Close";
```